src/.env.sample

```
@@ -156,3 +159,25 @@ APPLICATION_BASE_URL=/user
 SCHEDULER_PERIODIC_JOB_NAME_USER_MVIEWS="project_users_repeatable_view_job"
 #scheduler service job name to initialize the view and it is a one time job
 SCHEDULER_JOB_NAME_USER_MVIEWS="BuildMaterializedViewsprojectUsers"
+# Add default tenant code for existing users 
+DEFAULT_TENANT_CODE='DEFAULT'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)